### DISCUSSION TITLE
Piano discussion

### DISCUSSION THREAD

**Discussion by koko1928** [koko1928/Browser-piano#4] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by koko1928** [comment]
Hi

**Reply by mama1928** [reply]
hi

**Reply by koko1928** [reply]
> hi

nice

**Comment by mama1928** [comment]
Hello

**Comment by mama1928** [comment]
Why
